project_scrum/demo/project_scrum_demo.xml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <record id="base.user_demo" model="res.users">
+        <field name="groups_id" eval="[(4, ref('project.group_project_user')),
+        (4, ref('project.group_project_manager'))]"/>
+    </record>
+
+    <record id="project_project_1" model="project.project">
+        <field name="use_scrum">True</field>
+        <field name="date_start" eval="time.strftime('%Y-%m-01 10:00:00')"/>
+        <field name="name">Project Scrum</field>
+        <field name="color">3</field>
+        <field name="partner_id" ref="base.res_partner_10"/>
+       <field name="privacy_visibility">employees</field>
+        <field name="user_id" ref="base.user_demo"/>
+        <field name="description">Model for Testing Scrum using in projects</field>
+        <field name="alias_model">project.task</field>
+    </record>
+
+    <record id="project_project_2" model="project.project">
+        <field name="use_scrum">True</field>
+        <field name="date_start" eval="time.strftime('%Y-%m-01 10:00:00')"/>
+        <field name="name">Project Testing Scrum</field>
+        <field name="color">3</field>
+        <field name="partner_id" ref="base.res_partner_10"/>
+        <field name="privacy_visibility">employees</field>
+        <field name="user_id" ref="base.user_demo"/>
+        <field name="description">Model for Testing Scrum using in projects</field>
+        <field name="alias_model">project.task</field>
+    </record>
+
+    <record id="project_sprint_1" model="project.scrum.sprint">
+        <field name="name">Sprint 1</field>
+        <field name="project_id" ref="project_project_1"/>
+        <field name="date_start" eval="(DateTime.today() - relativedelta(weeks=1)).strftime('%Y-%m-%d')"/>
```

Comment:
With very long lines, put each attribute on a separate line. Now for instance I need to scroll horizontally to see what is in the eval.